.github/workflows/check-pr.yml

```diff
@@ -51,6 +51,81 @@ jobs:
         with:
           image: ghcr.io/podkrepi-bg/api/migrations:pr
 
+  build-migrations-docker-image:
+    name: Build migrations docker image
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Login to GitHub Packages Docker Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Get tag
+        if: ${{ startsWith(github.ref, 'refs/tags/v') }}
+        run: echo "VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+
+      - name: Get sha
+        if: ${{ !startsWith(github.ref, 'refs/tags/v') }}
+        run: echo "VERSION=master-$(echo $GITHUB_SHA | cut -c 1-6)" >> $GITHUB_ENV
+
+      - name: Build and push
+        uses: docker/build-push-action@v3
+        env:
+          NODE_ENV: production
+        with:
+          push: true
+          file: Dockerfile.migrations
+          target: base
+          build-args: |
+            APP_VERSION=${{ env.VERSION }}
+          tags: |
+            ghcr.io/podkrepi-bg/api/migrations:master
+            ghcr.io/podkrepi-bg/api/migrations:${{ env.VERSION }}
+
+      - name: Scan with Mondoo
+        uses: mondoohq/actions/docker-image@main
+        env:
+          MONDOO_CONFIG_BASE64: ${{ secrets.MONDOO_SECRET }}
+        with:
+          image: ghcr.io/podkrepi-bg/api/migrations:${{ env.VERSION }}
+
+  build-seed-docker-image:
```

Comment:
I guess this is the only thing that needs to be added but you should definitely remove the login and push. You need the same steps as the build for the migrations container. The only difference should be the target of the build